simplq/src/components/common/QueueHistory/HistoryRow.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import './QHistory.css';
+import moment from 'moment';
+import AddIcon from '@material-ui/icons/Add';
+import NotificationsIcon from '@material-ui/icons/Notifications';
+import DeleteIcon from '@material-ui/icons/Delete';
+import TouchAppIcon from '@material-ui/icons/TouchApp';
+import PauseIcon from '@material-ui/icons/Pause';
+import PlayArrowIcon from '@material-ui/icons/PlayArrow';
+import UpdateIcon from '@material-ui/icons/Update';
+
+export default (props) => {
+
+  const getMessage = () => {
+    var message = '';
+    console.log(props);
+    if (props.data.action === 'ADDED') {
+      message = props.data.name.trim () + ' was added';
+    } else if (props.data.action === 'REMOVED') {
+      message = props.data.name.trim () + ' was removed';
+    } else if (props.data.action === 'NOTIFIED') {
+      message = props.data.name.trim () + ' was notified';
+    } else if (props.data.action === 'JOINED') {
+      message = props.data.name.trim () + ' joined the queue';
+    } else if (props.data.action === 'PAUSED') {
+      message = 'The queue was paused';
+    } else if (props.data.action === 'RESUMED') {
+      message = 'The queue was resumed back';
+    } else if (props.data.action === 'UPDATED') {
+      message = 'Queue Settings was updated';
+    } else {
+      message = null;
+    }
+    return message;
+  };
+
+  const getIcon = () => {
+    return props.data.action === 'ADDED'
+      ? <AddIcon fontSize="medium" />
+      : props.data.action === 'REMOVED'
+          ? <DeleteIcon fontSize="medium" />
+          : props.data.action === 'NOTIFIED'
+              ? <NotificationsIcon fontSize="medium" />
+              : props.data.action === 'JOINED'
+                  ? <TouchAppIcon fontSize="medium" />
+                  : props.data.action === 'PAUSED'
+                      ? <PauseIcon fontSize="medium" />
+                      : props.data.action === 'RESUMED'
+                          ? <PlayArrowIcon fontSize="medium" />
+                          : props.data.action === 'UPDATED'
+                              ? <UpdateIcon fontSize="medium" />
+                              : null;
+  }
+
+  return (
+    <React.Fragment>
+      <div className="background-color-red history-row">
+        <div className="history-row-icon">
```

Comment:
After the import change, use className=styles["background-color-red history-row"]